test/integration/aws/default/build/flow_log.tf

```diff
@@ -0,0 +1,68 @@
+resource "aws_flow_log" "fl-vpc" {
```

Comment:
You might consider naming these more generically - alpha, beta, gamma, etc.  This make it easier to write tests later that are not interested in what the flow log is watching.  Ping me if that doesn't make sense :)